packages/paste-core/utilities/screen-reader-only/src/index.tsx

```diff
@@ -1,6 +1,11 @@
 import styled from '@emotion/styled';
 
-export const ScreenReaderOnly = styled.span<{}>`
+interface ScreenReaderOnlyProps {
+  // as prop isn't typed correctly from styled package https://github.com/emotion-js/emotion/issues/1137
+  as?: keyof JSX.IntrinsicElements;
+}
+
+export const ScreenReaderOnly = styled.div<ScreenReaderOnlyProps>`
```

Comment:
Ah crap, I need to swap this back to a span from my testing why the as prop wasn’t typing properly.